apps/zotonic_launcher/src/command/zotonic_cmd_addsite.erl

```diff
@@ -189,6 +192,10 @@ parse_args([ "-n", Schema | Args ], Acc) ->
     parse_args(Args, Acc#{ dbschema => Schema });
 parse_args([ "-a", Pw | Args ], Acc) ->
     parse_args(Args, Acc#{ admin_password => Pw });
+parse_args([ "-S", "true" | Args ], Acc) ->
```

Comment:
Maybe make sense to:
* Not handle for `"true"` value exactly in this place
* Handle `"TRUE"`/`"true"`/`1`/`"FALSE"`/`"false"`/`"0"`/"fAlSe"`/`"tRUe"` as `true`/`false`